internal/naming/names.go

```diff
@@ -325,6 +325,19 @@ func InstancePostgresDataVolume(instance *appsv1.StatefulSet) metav1.ObjectMeta
 	}
 }
 
+// TODO(benjaminjb): We need to consider shortening the PVC name
+// or at least need to document so that people can avoid too-long names
```

Comment:
🤔 What is too long? I think this is saying 253 characters? https://github.com/CrunchyData/postgres-operator/blob/8b14582aaa9df139b8eb38fe48a47f546db61f02/internal/naming/limitations.md?plain=1#L89

Reply:
Oh, yeah, I documented this (that it wouldn't likely be a problem because of the 253 character length), removing that TODO now